### ISSUE TITLE
Don't show home bar again after pressing a button

### ISSUE THREAD

**Issue by Simonx22** [OatmealDome/dolphin-ios#47]
https://user-images.githubusercontent.com/26326692/214348059-ec0ad7f2-0e14-401b-8c76-513355c372e9.MOV

**Comment by OatmealDome** [comment]
I agree not showing the indicator would be nice. Unfortunately, there doesn't seem to be an API for this.
